テクスチャの変更

creativival opened the pull request:
基本形状のテクスチャを変更できるようにした
仮想オブジェクトの転送のチェック済み
テストファイル同梱
![img_7309](https://user-images.githubusercontent.com/33368327/49338955-1d67a500-f66d-11e8-902d-9ed30e3c4ccc.PNG)

creativival committed:
テクスチャの変更

基本形状のテクスチャを変更できるようにした
仮想オブジェクトの転送のチェック済み

champierre commented:
@creativival レビューしやすいように、他のPRでマージされた変更分を取り除きました。
こういう場合 git rebase を使います。
実行した手順は以下の通りです。

```
➜  s2ar git:(master) git checkout feature/change_material
Switched to branch 'feature/change_material'
Your branch is up-to-date with 'origin/feature/change_material'.
➜  s2ar git:(feature/change_material) git rebase master # <-- rebase をおこないます
First, rewinding head to replay your work on top of it...
Applying: テクスチャの変更
➜  s2ar git:(feature/change_material) git push -f origin feature/change_material # <--  rebase をおこなったあとは force push をおこないます。force push は歴史を改ざんしてしまう操作なので、実行するときは注意してください。
Counting objects: 1, done.
Writing objects: 100% (1/1), 336 bytes | 0 bytes/s, done.
Total 1 (delta 0), reused 0 (delta 0)
To git@github.com:champierre/s2ar.git
 + 24a438b...4fc40a7 feature/change_material -> feature/change_material (forced update)
```

creativival committed:
テクスチャで地球の画像を選べるようにした

変更点
ViewController.swift, s2ar.js - earth を選択できるようにコードを追記
material フォルダに "earth.jpg" の画像を追加

creativival commented:
rebase ありがとうございます。テクスチャで地球の画像を選べるように小規模な修正を行いました。ご確認をお願いいたします。
